src/config.py

```diff
@@ -0,0 +1,16 @@
+from os import getenv
+
+debug = True
+
+if debug:
+    from dotenv import load_dotenv
+
+    load_dotenv(override=True)
```

Comment:
```suggestion
from dotenv import load_dotenv

load_dotenv(override=True)
```

Reply:
because i had copied your orig code. though it is an aways true, Edit: fixed